suite-native/receive/src/screens/ReceiveModalScreen.tsx

```diff
@@ -152,6 +163,7 @@ export const ReceiveModalScreen = () => {
                 <ReceiveModalScreenSubHeader
                     accountKey={accountKey}
                     tokenContract={tokenContract}
+                    closeActionType={closeActionType}
```

Comment:
this prop is redundant IMO, you can access it directly inside of the component with use of `useRoute` hook.

Reply:
Aha, that is a very good idea! 🚀

Reply:
ffd098bc78